src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -53,9 +51,59 @@ class HttpGoogleCloudStorageDAO(
 
   val system = akka.actor.ActorSystem("system")
 
-  override def createBucket(userInfo: UserInfo, projectId: String, workspaceId: String): Unit = {
-    val inserter = getStorage(getBucketCredential(userInfo)).buckets.insert(projectId, new Bucket().setName(getBucketName(workspaceId)))
-    retry(inserter.execute,when500)
+  override def createBucket(userInfo: UserInfo, projectId: String, workspaceId: String, workspaceName: WorkspaceName): Unit = {
+    val bucketName = getBucketName(workspaceId)
+    val groups = getGroupDirectory.groups
+    insertGroup(groups,newGroup(bucketName,workspaceName,WorkspaceAccessLevel.Read)) {
+      insertGroup(groups,newGroup(bucketName,workspaceName,WorkspaceAccessLevel.Write)) {
+        insertGroup(groups,newGroup(bucketName,workspaceName,WorkspaceAccessLevel.Owner)) {
+          val readersEntity = makeGroupEntityString(toGroupId(bucketName,WorkspaceAccessLevel.Read))
+          val writersEntity = makeGroupEntityString(toGroupId(bucketName,WorkspaceAccessLevel.Write))
+          val ownersEntity = makeGroupEntityString(toGroupId(bucketName,WorkspaceAccessLevel.Owner))
+          val userEntity = s"user-${userInfo.userEmail}"
+          val bucket = new Bucket().setName(bucketName).setAcl(List(
+              newBucketAccessControl(readersEntity,WorkspaceAccessLevel.Read),
+              newBucketAccessControl(writersEntity,WorkspaceAccessLevel.Write),
+              newBucketAccessControl(ownersEntity,WorkspaceAccessLevel.Owner),
+              newBucketAccessControl(userEntity,WorkspaceAccessLevel.Owner)
```

Comment:
the user should not be added directly to the bucket acls, they should be added to the owner group, same goes for default object acls below.

If we put them directly on the bucket and not in the group we won't see them as owning. Even if they are in the group as well, we will not be able to change their acl later.